src/js/components/molecules/list-section/list-section.scss

```diff
@@ -1,5 +1,7 @@
 /** @format */
 
+@import "~scss/settings/_variables.scss";
```

Comment:
we don't do this anywhere else… variables are just available aren't they?

Reply:
i thought we talked about doing this in order to create more independent components. since then i have been adding it to other components.

Reply:
We should choose a direction and stick with it. I'm fine with both.